.env

```diff
@@ -1,12 +1,2 @@
-RADICLE_DOMAIN=ENC[AES256_GCM,data:NSl+Sk6sbZEkFDtJN+uKFMozlFhWSA/pnXF4Sw==,iv:5ApC8DYO8DbgN5BctjidZoqswrrqE4pVqDiisOkRyTE=,tag:XMrC4TyBaKzsH8LQXkgmjg==,type:str]
-RADICLE_ORGS=ENC[AES256_GCM,data:HwCHyj+MWJ3+p6ZsFxgozMe8nFuTyrNtQcMsGVa41nPrs8NwJT6lBEK5,iv:7UjX6FHDHIz7fvCiXeZ8cGy0Q7m+R0z3/nGyP7WJ6ys=,tag:i4UtVD263wd/b59vOW9sHw==,type:str]
-ETH_RPC_URL=ENC[AES256_GCM,data:xpEgi+7V0R5m9QCZcMlK7kbe/VVI+PLHHxBXHHCIeO8xDI2dYlCjxv9lDQlXB3N3i0VUkvt/W0Zj7f0tHu/05hOC6Q==,iv:5+AH2YWPucl6Jcl7QWhodGniqHMveR8kcP6snEdch7I=,tag:+6Hq9X+9vxVPHeWHcGrdug==,type:str]
-RADICLE_INFLUXDB_PASSWORD=ENC[AES256_GCM,data:QIR5VbHNnXpqGAQtdbobqfAAfPY=,iv:EwmqGnV3ZAAbqqyxfygTeu+8k4bv/0f2UONalEsD4Z4=,tag:7XsAqdy1F2DlngzSurO04A==,type:str]
-RADICLE_INFLUXDB_TOKEN=ENC[AES256_GCM,data:19NKQrxIdCztW4mLh5/GfxZtAK4SBwYM2BRz/LSjFi2LPFYDuRITm/6R9DQ53va0aySG6ARM49AVcutOQz/Q94Sha21U2AmsHGvxs0cTYNDGMx+BQJcvww==,iv:XqIpHtSMXbjbD9EKUFDDkX2awpz4xFmCwL47Nx1+wVA=,tag:EEcg6E4CbmWAHyQK3bhRuw==,type:str]
-sops_gcp_kms__list_0__map_enc=CiQAw9HjBpHvRP7Kqe3v7vDzWKB1cRZGkGj78ZTwANxZG4iQX9USSQCYOiXnY7hYrqHYQnB51Y1kYph98xPFqD8gbwccE3OuVxkroLxd30FFTVUBcxxM+sykv4JDbv9VkkG5Dsm3zc+P4WhTsrxPl1s=
-sops_version=3.7.1
-sops_unencrypted_suffix=_unencrypted
-sops_gcp_kms__list_0__map_resource_id=projects/radicle-services/locations/global/keyRings/sops/cryptoKeys/sops-key
-sops_gcp_kms__list_0__map_created_at=2021-10-29T15:15:04Z
-sops_lastmodified=2021-10-29T15:15:04Z
-sops_mac=ENC[AES256_GCM,data:jD3xSgSAKzDg+HnM4h96/G51zB8YI3KPVCOENh2jiiwm9/m+p9+vcE8J9UXDO3mIEVfMy6WaniFgEn3g2eDqoxIDXij251rchzpaSNfwynUimDC9+3jibUpnD0fOe2Ky/1awd/dt/DC0kPaTG8u5cgM2FkuRSvxTGM0MF/wk5v0=,iv:KCvjnUo+qkFid2DHzpWy2fnXeVGq8vtr+VtiuoMMweY=,tag:p3aRTn06H6iIDB0yCfDYlg==,type:str]
+RADICLE_DOMAIN=
+GIT_SERVER_ENV_SPECIFIC_OPTS=
```

Comment:
Can we call this `RADICLE_GIT_SERVER_OPTS`? I think it's clear that it's an env var already, but not that it's radicle-specific.